CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Changelog for NeoFS Node
 - tzhash library to 1.7.0 (#2348)
 - `github.com/hashicorp/golang-lru` to `v2.0.2`
 - `neofs-sdk-go` to `v1.0.0-rc.8`
+- BoltDB to 1.3.7
```

Comment:
What do you think about attaching the full source link? To be precise, the fork package is named bbolt.

Reply:
`BoltDB` is somewhat more humane, but let's try it this way.